tests/unit/model/test_model_config_init.py

```diff
@@ -0,0 +1,66 @@
+from unittest.mock import patch
+
+import pytest
+
+from zenml.enums import ModelStages
+from zenml.model import ModelConfig
+
+
+@pytest.mark.parametrize(
```

Comment:
See comment above - why is it important that this configuration gives a warning?

Reply:
removed